code-scanning/anchore.yml

```diff
@@ -20,8 +20,14 @@ on:
   schedule:
     - cron: $cron-weekly
 
+permissions:
+  contents: read
+
```

Comment:
Is this necessary given that the `Anchore-Build-Scan` job below has the `contents: read` permission?

Reply:
@kzantow thanks for checking it out, you can see [this](https://github.com/actions/starter-workflows/pull/1348#discussion_r792966337) for nice explanation by @laurentsimon , on why it is necessary. thanks again

Reply:
We've been merging similar 'defensive' changes to other workflows, so I'm going to go ahead with this one. 👍